SeleniumTestXUnit/Tests/Features/Item/Create.feature

```diff
@@ -0,0 +1,21 @@
+Feature: Create an Item in a Project
+As a logged user, the user wants to create an item in a project so that he can keep track of my tasks.
+
+Background:
+    Given the user is logged in
+        And the user has an existing project
+
+    @Draft
+    @Acceptance
+    Scenario: Create an Item Successfully
+        Given the user has selected a project
+        When the user clicks on the Add New Todo
+        Then the user enters an item name
```

Comment:
This should be an And step, however, steps on lines 12 and 13 should be done on one single step that click and sends keys to an input field.